docs/data/date-pickers/fields/ControlledSelectedSections.tsx

```diff
@@ -3,17 +3,15 @@ import Stack from '@mui/material/Stack';
 import Button from '@mui/material/Button';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
-import { MuiDateSectionName, FieldSelectedSections } from '@mui/x-date-pickers';
+import { FieldSectionType, FieldSelectedSections } from '@mui/x-date-pickers';
 import { DateField } from '@mui/x-date-pickers/DateField';
 
 export default function ControlledSelectedSections() {
   const [selectedSections, setSelectedSections] =
     React.useState<FieldSelectedSections>(null);
   const inputRef = React.useRef<HTMLInputElement>(null);
 
-  const setSelectedDateSectionName = (
-    selectedDateSectionName: MuiDateSectionName,
-  ) => {
+  const setSelectedDateSectionName = (selectedDateSectionName: FieldSectionType) => {
```

Comment:
Nitpick, but this example becomes a bit out of sync after the type rename. 🤔 🙈

Reply:
Nice catch :ok_hand: